src/Templates/NewModule/build.psake.ps1

```diff
@@ -0,0 +1,565 @@
+##############################################################################
+# PREVIEW VERSION OF PSAKE SCRIPT FOR MODULE BUILD & PUBLISH TO THE PSGALLERY
+##############################################################################
+#
+# We are hoping to add support for publishing modules to the PowerShell gallery
```

Comment:
Probably need to update the comments here to remove the language about our plans